src/lister_haxe/test/lister_test.hx

```diff
@@ -0,0 +1,36 @@
+package test;
+
+import haxe.unit.TestCase;
+using StringTools;
+using test.TestCaseExtender;
```

Comment:
I wrote a little extension method for the native `haxe.unit.TestCase` class.  The `using` here is pulling that in, looking for it to be in the `test` package (e.g. namespace / folder).  See http://haxe.org/manual/type-system-modules-and-paths.html